test/screencast.spec.ts

```diff
@@ -163,6 +163,7 @@ class VideoPlayer {
 
 describe('screencast', suite => {
   suite.slow();
+  suite.flaky();
```

Comment:
Let's mark only tests that analyse pixels, the rest are passing AFAICT.

Reply:
I'll let you re-enable them.